include/links-model.php

```diff
@@ -48,9 +48,24 @@ public function __construct( &$model ) {
 
 		$this->home = home_url();
 
-		// Hooked with normal priority because it needs to be run after static pages is set in language data.
+		// Hooked with normal priority because it needs to be run after static pages is set in language data. Must be done early (before languages objects are created).
 		add_filter( 'pll_additional_language_data', array( $this, 'set_language_home_urls' ), 10, 2 );
 
+		if ( did_action( 'pll_init' ) ) {
+			$this->init();
+		} else {
+			add_action( 'pll_init', array( $this, 'init' ) );
+		}
+	}
+
+	/**
+	 * Initializes links.
+	 *
+	 * @since 3.5
+	 *
+	 * @return void
+	 */
+	public function init() {
```

Comment:
Based on the comment above, we should go back and let this method do nothing.
```suggestion
	/**
	 * Initializes the links model.
	 * Does nothing by default.
	 *
	 * @since 3.5
	 *
	 * @return void
	 */
	public function init() {}
```